docs/firefly/verify_download.md

```diff
@@ -138,7 +139,7 @@ To follow these instructions you need the [Xcode Command Line Tools](https://www
 
 #### Prerequisites
 
-To follow these instructions you need the `sha256sum` package, which is included with most Linux distributions.
+You will need the `sha256sum` package, which is included with most Linux distributions, to complete these instructions.
```

Comment:
I'd say use simple present and also unwrap the clause into a separate sentence.

Reply:
I don't think it needs a separate sentence; its pretty much a "same breath" kind of deal.

Reply:
You could remove the "to complete these instructions" as it's kind of implied.

Reply:
Was overruled here👌